target/linux/ramips/image/mt7620.mk

```diff
@@ -193,6 +193,15 @@ define Device/gl-mt300a
 endef
 TARGET_DEVICES += gl-mt300a
 
+define Device/u25awf
+  DTS := U25AWF
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
`ralink_default_fw_size_16M` is 16121856 bytes (0xf60000) which doesn't match the size if your firmware partition. Please use 16064k as IMAGE_SIZE.